pkgs/development/libraries/prometheus-cpp/default.nix

```diff
@@ -32,6 +32,19 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
+  # https://github.com/jupp0r/prometheus-cpp/issues/587
+  postPatch = ''
+    substituteInPlace cmake/prometheus-cpp-core.pc.in \
+      --replace '$'{exec_prefix}/@CMAKE_INSTALL_LIBDIR@ @CMAKE_INSTALL_FULL_LIBDIR@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_INCLUDEDIR@ @CMAKE_INSTALL_FULL_INCLUDEDIR@
+    substituteInPlace cmake/prometheus-cpp-pull.pc.in \
+      --replace '$'{exec_prefix}/@CMAKE_INSTALL_LIBDIR@ @CMAKE_INSTALL_FULL_LIBDIR@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_INCLUDEDIR@ @CMAKE_INSTALL_FULL_INCLUDEDIR@
+    substituteInPlace cmake/prometheus-cpp-push.pc.in \
+      --replace '$'{exec_prefix}/@CMAKE_INSTALL_LIBDIR@ @CMAKE_INSTALL_FULL_LIBDIR@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_INCLUDEDIR@ @CMAKE_INSTALL_FULL_INCLUDEDIR@
```

Comment:
```suggestion
    for file in cmake/prometheus-cpp-core.pc.in cmake/prometheus-cpp-pull.pc.in cmake/prometheus-cpp-push.pc.in; do
      substituteInPlace $file \
        --replace '$'{exec_prefix}/@CMAKE_INSTALL_LIBDIR@ @CMAKE_INSTALL_FULL_LIBDIR@ \
        --replace '$'{prefix}/@CMAKE_INSTALL_INCLUDEDIR@ @CMAKE_INSTALL_FULL_INCLUDEDIR@
    do
```

Reply:
Switched to upstream fix in https://github.com/NixOS/nixpkgs/pull/172150/commits/248292d15172034ab28b03b0d2e685848960323c + https://github.com/NixOS/nixpkgs/pull/172150/commits/3688d1b4397e1b25c26bae4368810d533a779d6c, so should be fine now.